app/Http/Controllers/RequestController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\OrderRequest;
+use App\Models\Author;
+use App\Models\Book;
+use App\Models\Category;
+use App\Models\Request;
+use App\Models\User;
+use Auth;
+use Carbon\Carbon;
+
+class RequestController extends Controller
+{
+    public function index()
+    {
+        $requests = Request::with('user')->where('user_id', Auth::id())->paginate(5);
```

Comment:
```suggestion
        $requests = Auth::user()->request()....
```
viết như này cho ngắn gọn